### PR TITLE
code-server / VSCode for power9

### PR DESCRIPTION
**Pull request by afraz-ahmed**
For INC1034947

Update for #187 

VSCode-3.4.0-GCCcore-8.3.0.eb
* [x] Assigned to reviewer
* [ ] EL7-power9

### CODE REVIEW

**Comment by afraz-ahmed** [comment]
nodejs/12.6.1-GCCcore-8.3.0 has been merged from #187

File: easybuild/easyconfigs/v/VSCode/VSCode-3.4.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+]
+
 dependencies = [
     ('nodejs', '12.16.1'),
 ]
 
-preinstallopts = 'cd %(name)s-%(version)s &&'
-install_cmd = 'npm install --prefix %(installdir)s --global %(name)s'
+preinstallopts = 'cd code-server-%(version)s &&'
+install_cmd = 'npm install --prefix %(installdir)s -global code-server'
```
Comment (branfosj) [inline easybuild/easyconfigs/v/VSCode/VSCode-3.4.0-GCCcore-8.3.0.eb:25]: Does the `global` command need an extra `-`, i.e. `--global`?

Comment (afraz-ahmed) [reply easybuild/easyconfigs/v/VSCode/VSCode-3.4.0-GCCcore-8.3.0.eb:25]: It works with a single `-`. It was `-g` previously, but James said he prefers to use full names when possible (`-global`).

**Comment by orbsmiv** [comment]
Built on all requested platforms

**Pull request merged**